### ISSUE TITLE
Did the DSOD model code have tensorflow  version?

### ISSUE THREAD

**Issue by Tzpchris** [szq0214/DSOD#26]
I want to use TensorFlow to implement this DSOS, I try to write, but some mistakes. So are there any TensorFlow versions？
Thanks

**Comment by nszceta** [comment]
I found that Tensorflow/Keras usually gives worse accuracy than Caffe. So in this case it might be better to stick with Caffe https://github.com/fchollet/keras/issues/4444

**Comment by Windaway** [comment]
https://github.com/Windaway/DSOD-Tensorflow
DSOD TF撒ver手without不any管support。
